### ISSUE TITLE
Right click context menu not being created in Windows 11 24H2

### ISSUE THREAD

**Issue by passionforpins** [vpinball/b2s-backglass#127]
**Describe the bug**
Running b2sserverregisterapp to add the context menu no longer seems to work. Right clicking on a vpx file no longer shows the create tablename.res option.
However, right clicking on a directb2s file DOES show the context menu- but this is of no use

This only happened after a windows feature update to 24H2 

**Versions & Settings**
 - VPX: 10.8.0 RC5 rev 2035
 - VPM: 3.6.0 1167
 - B2S: 2.5.0 320
 - Windows 11 Home 24H2

**Comment by JockeJarre** [comment]
Thanks. I do not have 24H2 yet... M$ hasn't offered it to me... ;)
We'll see once it is here.

**Comment by Retsamikit** [comment]
Got 24h2 yesterday...
For me the complete b2s-context menu is gone. Can´t see an entry anymore. Looking in CCleaner: Extras/Autostart/Contextmenu its also gone.

**Comment by JockeJarre** [comment]
Hi, from our discussions online, can we close this issue now?

**Comment by passionforpins** [comment]
Why close? Is the right click context menu registering now? Some new version of b2s server I'm not aware of? I had to manually add the entries back in.

**Comment by JockeJarre** [comment]
Yes, they should be there now, but you right click the b2s file instead. It was confirmed by retsamkit on discord, but if you say it's not working, I will have another look.

**Comment by passionforpins** [comment]
ok I see. I haven't been on discord since last fall when my brother passed. I might have seen the discussion if it was on vpu.
I still MUCH prefer right clicking the vpx file as there are many times I do not have or want a b2s but still need the tablename.res.

**Comment by JockeJarre** [comment]
Actually a if a table is called 

"Tablename (Author date) someone mod.vpx" 

and fuzzy search is turned on, then I propose that it should search the res file in this order:

1. "Tablename (Author date) 3scr.res" (as the backglass)
2. "Tablename (Author date) someone mod.res" (as the table)

**Comment by Heartman30044** [comment]
Sounds reasonable.
